resources/windows/wix/customactions.wxs.erb

```diff
@@ -242,19 +242,23 @@ Sub ExecuteCommand (Command)
 
   Log "Executing Command : " & Command, False
 
-  ' https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/windows-scripting/2f38xsxe%28v%3dvs.84%29
-  ' Exec rather than Run to be able to capture stdout
-  Dim wshScriptExec : Set wshScriptExec = wshShell.Exec(Command)
-
-  Do While Not wshScriptExec.StdOut.AtEndOfStream
-    Log wshScriptExec.StdOut.ReadLine(), False
+  Dim tempFilePath : tempFilePath = fso.GetTempName()
+  ' https://docs.microsoft.com/en-us/previous-versions/windows/internet-explorer/ie-developer/windows-scripting/d5fk67ky%28v%3dvs.84%29
+  ' intWindows Style - 0 - Hides the window and activates another window.
+  ' bWaitOnReturn - True - waits for program termination
+  Dim exitCode : exitCode = wshShell.Run(Command & " 2>&1 > """ & tempFilePath & """", 0, True)
+
+  Dim outFile : Set outFile = fso.OpenTextFile(tempFilePath)
```

Comment:
DO you need error detection around this or just silently fail?

Reply:
I think we're OK without anything for now, since I wouldn't expect we'd fail to open our randomly created file ever. But it's something that could be improved in a subsequent PR.